.github/workflows/conformance.yml

```
@@ -17,57 +17,72 @@ jobs:
-        pull: true
-        cache-from: type=registry,ref=opengisch/signalo-django:latest
-        cache-to: type=registry,ref=opengisch/signalo-django:latest,mode=max
-        tags: opengisch/signalo-django:latest
+      - name: Build Django image
+        uses: docker/build-push-action@v4
+        with:
+          context: .
+          provenance: false
+          file: docker/django/Dockerfile
+          push: ${{ github.repository == 'opengisch/django-ogcapif' }}
+          pull: true
+          cache-from: type=registry,ref=opengisch/signalo-django:latest
+          cache-to: type=registry,ref=opengisch/signalo-django:latest,mode=max
+          tags: opengisch/signalo-django:latest
 
-    - name: Do quickstart
-      run: |
-        # copy default conf
-        cp .env.example .env
+      - name: Do quickstart
+        run: |
+          # copy default conf
+          cp .env.example .env
 
-        # start the stack
-        docker compose up --build -d
+          # start the stack
+          docker compose up --build -d
 
-        # deploy static files and migrate database
-        docker compose exec django python manage.py collectstatic --no-input
-        docker compose exec django python manage.py migrate --no-input
-        docker compose exec django python manage.py populate_vl
-        docker compose exec django python manage.py populate_signs_poles
+          # deploy static files and migrate database
+          docker compose exec django python manage.py collectstatic --no-input
+          docker compose exec django python manage.py migrate --no-input
+          docker compose exec django python manage.py populate_vl
+          docker compose exec django python manage.py populate_signs_poles
 
-    - name: Healthcheck
-      run: wget --no-check-certificate https://localhost/oapif/collections/signalo_core.pole/items
+      - name: Healthcheck
+        run: wget --no-check-certificate https://localhost/oapif/collections/signalo_core.pole/items
 
-    - name: Run conformance test suite
-      run: docker compose run conformance_test
+      - name: Run conformance test suite
+        run: docker compose run conformance_test
 
-    - name: Archive production artifacts
-      uses: actions/upload-artifact@v3
-      with:
-        name: emailable-report
-        path: |
-          ./_test_outputs/**/emailable-report.html
+      - name: Archive production artifacts
+        uses: actions/upload-artifact@v3
+        with:
+          name: emailable-report
+          path: |
+            ./_test_outputs/**/emailable-report.html
 
-    - name: Failure logs
-      if: failure()
-      run: docker-compose logs
+      - name: Check baseline
+        id: baseline
+        continue-on-error: true
+        run: |
+          pip install lxml
+          python scripts/parse_report.py _test_outputs/emailable-report.html baseline.json
+          exit_code=$?
+          echo "exit_code=$exit_code" >> $GITHUB_OUTPUT
```

```suggestion
          EXIT_CODE=0
          python scripts/parse_report.py _test_outputs/emailable-report.html baseline.json  || EXIT_CODE=$?
          echo "exit_code=${EXIT_CODE}" >> $GITHUB_OUTPUT
          [[ ${EXIT_CODE} -eq 1 ]] && exit 1 || :
```

This should solve our problem: only error on 1